src/Backend/Modules/Extensions/Engine/Model.php

```diff
@@ -505,11 +505,12 @@ public static function getModuleInformation($module)
      */
     public static function getModules()
     {
-        $installedModules = (array) BackendModel::getContainer()->get('database')->getRecords(
-            'SELECT name FROM modules',
-            null,
-            'name'
-        );
+        $em = BackendModel::get('doctrine.orm.entity_manager');
+
+        $allInstalledModules = $em->getRepository('\Backend\Core\Entity\Module')->findAll();
```

Comment:
This could be done cleaner with a DQL query that fetches scalar results (so just the name, not the full objects.